android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldRadioView.java

```diff
@@ -160,6 +160,15 @@ public void setParentView(View view) {
 
     @Override
     public void setUpdateMode(boolean enabled) {
+        List<Option> availableOptions = field.getAvailableOptions();
+
+        if (availableOptions != null) {
+            for (Option opt : availableOptions) {
+                RadioButton radioButton = findViewWithTag(opt);
```

Comment:
We should avoid searching the views all the time, as it's in a static context we could keep the views indexed in some structure.

Reply:
A hashmap maybe, but be aware to not leak memory